configure.ac

```diff
@@ -499,6 +499,7 @@ AS_IF([test "x$with_libsystemd" = "xyes"], [
               [with_systemdsystemunitdir=$($PKG_CONFIG --variable=systemdsystemunitdir systemd)])
   AS_IF([test "x$with_systemdsystemunitdir" != "xno"], [
     AC_SUBST([systemdsystemunitdir], [$with_systemdsystemunitdir])
+    AC_DEFINE_UNQUOTED([SYSTEM_DATA_UNIT_PATH], ["$with_systemdsystemunitdir"], ["unit path"])
```

Comment:
I think you want this outside the `AS_IF` since otherwise `SYSTEM_DATA_UNIT_PATH` might not be defined. Building in a (maybe too minimal) container for #2379:
```
../src/libostree/ostree-impl-system-generator.c: In function 'require_internal_units':
../src/libostree/ostree-impl-system-generator.c:131:18: error: 'SYSTEM_DATA_UNIT_PATH' undeclared (first use in this function)
```
Or maybe you need some `#ifdef` in the code or not to build the generator in this case.

Reply:
I filed #2380 to fix this.